### ISSUE TITLE
defadvice in dbgp.el breaks open-network-stream in emacs 24.5 and 25.1

### ISSUE THREAD

**Issue by ahungry** [itorres/geben-on-emacs#17]
The issue is on line 669 of dbgp.el and the error caused is:

`ad-Advice-open-network-stream: Wrong type argument: processp, nil`

when many of the emacs network functions (gnus send mail, list-packages etc. are called).

It only occurs after a manual (require 'geben) - if you just install it from MELPA the issue does not seem to occur.

**Comment by itorres** [comment]
Hi, where did you get your geben copy from? This repo is private and not published to melpa.

If you check #14 you'll see that this issue is a duplicate and fixed by 967bb64.

**Comment by ahungry** [comment]
Sorry - this was the first hit when searching for it (the original geben link was on Google code, and the headers in the geben file still point there, which is a 404 since google code went down).

I'll look into where the melpa source comes from and see if I can get 967bb64 applied there if it isn't at the moment.

**Comment by ahungry** [comment]
So, it turns out the MELPA recipe still points to the old google code svn trunk (somehow that's still up, even though the project page is not).

Since I'm doubtful that will be the case indefinitely, I'd like to submit this repository (or my own fork of it) to MELPA as the replacement, so MELPLA users can receive your fix from February on the defadvice.

Did you intend to keep this repository private and off of MELPA?  If so, I am happy to take over maintenance of it in the interim (I will have the page at https://github.com/ahungry/geben - I removed the fork association to this repo to protect your privacy if desired).

**Comment by itorres** [comment]
I am a complete newbie concerning elisp and so it will probably be better maintained by you. I simply applied the patch from the mailing list.

When I said private I really meant "not publicized". Either way if you will be the maintainer of geben on MELPA it makes sense for your repo to be the upstream.

And no problem at all. _I_ am sorry if I sounded harsh, I was simply curious about how you ended here, I didn't expect to have an issue filled :)
